ckanext/datarequests/validator.py

```diff
@@ -20,7 +20,7 @@
 import datetime
 
 import ckan.plugins.toolkit as tk
-from ckanext.datarequests import db, common, constants
+from ckanext.datarequests import db, common, constants  # noqa: F401
```

Comment:
Instead of ignore the liniting issue why do you not remove the unused report db to fix the liniting issue?